lms/static/js/student_account/views/LoginView.js

```diff
@@ -103,12 +103,13 @@
             },
 
             saveError: function( error ) {
+                var msg = error.responseText;
                 if (error.status === 0) {
-                    this.errors = ['<li>' + gettext('Please check your internet connection and try again.') + '</li>'];
-                }
-                else {
-                    this.errors = ['<li>' + error.responseText + '</li>'];
+                    msg = gettext('Please check your internet connection and try again.');
+                } else if(error.status === 500){
+                    msg = gettext("Something went wrong.");
```

Comment:
An error has occurred. Try refreshing the page, or check your Internet connection.